lib/core/form/components/inplace-mat-form-field-wrapper/inplace-mat-form-field-wrapper.component.ts

```diff
@@ -0,0 +1,35 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+    ChangeDetectionStrategy,
+    Component,
+    Input,
+    ViewEncapsulation
+} from '@angular/core';
+
+@Component({
+    selector: 'adf-inplace-mat-form-field-wrapper',
+    templateUrl: './inplace-mat-form-field-wrapper.component.html',
+    styleUrls: ['./inplace-mat-form-field-wrapper.component.scss'],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Alfresco/alfresco-ng2-components/pullRequest?prid=8947659)
```suggestion
    styleUrls: ["./inplace-mat-form-field-wrapper.component.scss"],
```